addons/hr_holidays/static/src/persona_model_patch.js

```diff
@@ -19,7 +20,7 @@ export function getOutOfOfficeDateEndText(datetime) {
 patch(Persona.prototype, {
     /** @returns {string} */
     get outOfOfficeDateEndText() {
-        if (!this.main_user_id?.leave_date_to) {
+        if (!this.main_user_id?.leave_date_to || this.main_user_id.id === user.userId) {
```

Comment:
`this.main_user_id?.id === user.userId` is equivalent to `this.store.self.eq(this.main_user_id)`, this syntax is best since you avoid import and it follows best reactivity model of client-side code.